pkgconfig/mbedcrypto.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix="@CMAKE_INSTALL_PREFIX@"
+exec_prefix="${prefix}"
+libdir="${exec_prefix}/@CMAKE_INSTALL_LIBDIR@"
+includedir="${prefix}/include"
```

Comment:
These two lines have two issues:

1. `CMAKE_INSTALL_LIBDIR` isn't guaranteed to be a relative path, so you should use a proper path joining function from CMake code and use its result as the value for `libdir`. You can find a full example here: <https://github.com/simdjson/simdjson/pull/1767>
2. Instead of hardcoding `include`, you should use `CMAKE_INSTALL_INCLUDEDIR`, and then do the same as I suggested for `libdir`

The same applies to the other two `.pc.in` files, of course :)

Reply:
@Tachi107 are you proposing for item 2:
```
includedir="${prefix}/@{CMAKE_INSTALL_INCLUDEDIR}"
```
Obviously updated with a path join function.

Reply:
@Tachi107 suggestion is done, I also added a patch on top to use cmake builtin cmake_path, but I think we would need to bump `cmake_minimum_required(VERSION 3.5.1)` to `cmake_minimum_required(VERSION 3.20)` and I don't know if they will find that tenable.